edflow/iterators/tf_evaluator.py

```diff
@@ -101,5 +102,8 @@ def initialize(self, checkpoint_path=None):
             )
             self.hooks += [waiter]
 
+    def define_graph(self):
```

Comment:
did I get it right that we need this now that we moved the restore variables to initialize?

If so, we should really have a test that checks if our evaluator implementations are right.